### PR TITLE
New Zernike3D programs

### PR DESCRIPTION
**Pull request by DavidHerreros**
Update of Zernike3D protocols:

- Most of the Zernike3D protocols and viewers have been deleted from `scipion-em-xmipp` and moved to the new `scipion-em-flexutils` Plugin

### CODE REVIEW

File: xmipp3/protocols/protocol_apply_zernike3d.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+
+# **************************************************************************
+# *
+# * Authors:     David Herreros Calero (dherreros@cnb.csic.es)
```
Comment (jamesmkrieger) [inline xmipp3/protocols/protocol_apply_zernike3d.py:4]: probably I should be here too:
James Krieger (jmkrieger@cnb.csic.es)

Comment (DavidHerreros) [reply xmipp3/protocols/protocol_apply_zernike3d.py:4]: Added!

File: xmipp3/viewers/viewer_pdb_deform_zernike3d.py [hunk 2]
```diff
@@ -30,60 +30,80 @@
...
         form.addSection(label='Show deformation')
+        form.addParam('pdbIdChoice', params.EnumParam,
+                      condition='self.have_set==True',
+                      choices=list(self.deformed.getIdSet()),
+                      default=0,
+                      label='Structure to display', display=params.EnumParam.DISPLAY_COMBO,
+                      help='Select which structure to display from the IDs of the set')
         form.addParam('doShowPDB', params.LabelParam,
-                      label="Display original and deformed PDB")
+                      label="Display original and deformed PDB or volume")
```
Comment (jamesmkrieger) [inline xmipp3/viewers/viewer_pdb_deform_zernike3d.py:55]: This shouldn't say this as we ended up making this viewer only for atomic structures.

File: xmipp3/viewers/viewer_pdb_deform_zernike3d.py [hunk 3]
```diff
@@ -30,60 +30,80 @@
...
         form.addParam('doShowMorph', params.LabelParam,
-                      label="Display a morphing between the original and deformed PDB")
+                      label="Display a morphing between the original and deformed PDB or volume")
```
Comment (jamesmkrieger) [inline xmipp3/viewers/viewer_pdb_deform_zernike3d.py:57]: same here

File: xmipp3/viewers/viewer_volume_deform_zernike3d.py [hunk 4]
```diff
@@ -32,30 +32,60 @@
...
+        self.have_set = isinstance(self.deformed, Set)
+    else:
+        self.have_set = False
+
     def _defineParams(self, form):
         form.addSection(label='Show deformation')
+        form.addParam('idChoice', params.EnumParam,
+                      condition='self.have_set==True',
+                      choices=list(self.deformed.getIdSet()), default=0,
+                      label='Structure to display', display=params.EnumParam.DISPLAY_COMBO,
```
Comment (jamesmkrieger) [inline xmipp3/viewers/viewer_volume_deform_zernike3d.py:60]: This should be volume not structure

**Review by jamesmkrieger — CHANGES_REQUESTED** [review]
Everything looks good to me except for a few minor documentation things

**Review by jamesmkrieger — APPROVED** [review]

**Review by cossorzano — APPROVED** [review]

**Pull request merged**